EIPS/eip-6963.md

```diff
@@ -55,15 +56,14 @@ Each wallet provider will be announced with the following interface `EIP6963Prov
  */
 interface EIP6963ProviderInfo {
   uuid: string;
+  rdns: string;
```

Comment:
This makes it optional for TS interfaces

```suggestion
  rdns?: string;
```

And we should also move this one below "name" and "icon" in the interface

Reply:
agree moving the optional property to the end would be clearer, would do in both places

Reply:
Approving, but I'm hesitant that we should actually be defining the interface with a specific programming language. Unfortunately, it's too much of a headache to get the rendering working properly with something like [WebIDL](https://developer.mozilla.org/en-US/docs/Glossary/WebIDL) so this will have to be "good enough".